docs/sdk/guides/04-call-smart-contract-remotely.md

```diff
@@ -4,170 +4,85 @@ sidebar_position: 4
-
-1. Prepare your Dapp contract for both source chain and target chain
-
-    ```js
-    // deploy on the target chain first, then deploy on the source chain
-    contract RemoteTransactDemo is PangolinXApp {
-        constructor() public {
-            init();
-        }
-
-        uint256 public number;
-
-        ///////////////////////////////////////////
-        // used on the source chain
-        ///////////////////////////////////////////
-        function callAddOnTheTargetChain() public payable {
-            // 1. prepare the call that will be executed on the target chain
-            PalletEthereum.TransactCall memory call = PalletEthereum.TransactCall(
-                // the call index of substrate_transact
-                0x2902,
-                // the evm transaction to transact
-                PalletEthereum.buildTransactionV2(
-                    0, // evm tx nonce, nonce on the target chain + pending nonce on the source chain + 1
-                    1000000000, // gasPrice, get from the target chain
-                    600000, // gasLimit, get from the target chain
-                    0x50275d3F95E0F2FCb2cAb2Ec7A231aE188d7319d, // <------------------ change to the contract address on the target chain
-                    0, // value, the only allowed value here is 0
-                    hex"1003e2d20000000000000000000000000000000000000000000000000000000000000002" // the add function bytes that will be called on the target chain, add(2)
-                )
-            );
-            bytes memory callEncoded = PalletEthereum.encodeTransactCall(call);
-
-            // 2. Prepare the message payload
-            MessagePayload memory payload = MessagePayload(
-                28110, // spec version of target chain <----------- This may be changed, go to https://pangoro.subscan.io/runtime get the latest spec version
-                2654000000, // call weight
-                callEncoded // call encoded bytes
-            );
-
-            // 3. Send the message payload to the pangolin through the lane id
-            bytes4 laneId = 0;
-            uint64 _messageNonce = sendMessage(toPangolin, laneId, payload);
-        }
-
-        ///////////////////////////////////////////
-        // used on the target chain
-        ///////////////////////////////////////////
-        function add(uint256 _value) public {
-            // this 'require' makes this function only be called by the dapp contract on the source chain
-            require(
-                msg.sender == deriveSenderFromRemote(),
-                "msg.sender must equal to the address derived from the message sender address on the source chain"
-            );
-            number = number + _value;
-        }
+pragma solidity ^0.8.9;
+
+contract Callee {
+    uint256 public sum;
+
+    function add(uint256 _value) external {
+        sum = sum + _value;
     }
-    ```
+}
+```
+
+Deploy it on the Pangolin Smart Chain.
```

Comment:
Strongly recommend replacing all `Deploy` with web3 js script. The reader can create their first cross-chain dapper much easier than now. Only copy-paste and then checking the result is the best practice for such a tutorial.